library.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "RF24",
+  "keywords": "rf, radio, wireless, spi",
+  "description": "Optimized High Speed Driver for nRF24L01(+) 2.4GHz Wireless Transceiver",
+  "repository":
+  {
+    "type": "git",
+    "url": "https://github.com/TMRh20/RF24.git"
+  },
+  "include": [
+    "*.h",
+    "*.c",
+    "*.cpp",
+    "arch/ATTiny/*",
```

Comment:
Shouldn't we list here the dirs for the other architectures (BBB, RPi, ... ) too?